docs/src/man/syntax.md

```diff
@@ -284,6 +284,7 @@ page. Currently recognised keys:
 - `Description`: a page-specific description that gets displayed in search engines and
   link previews. Overrides the site-wide description in [`makedocs`](@ref). 
 - `Draft`: boolean for overriding the global draft mode for the page.
+- `DocStringsCollapsed`: if `true`, collapse all docstrings after loading the page (experimental).
```

Comment:
Marked as "experimental" so we can still change this even after the PR is merged/released. For example, https://github.com/JuliaDocs/Documenter.jl/pull/2394#issuecomment-1892874802

Reply:
Eh, yea, let's just keep it on the top-level. Doing namespacing adds other kinds of complexity. Some of these options will just be ignored by some backends.

```suggestion
- `DocStringsCollapsed`: on output formats that support this (i.e. HTML), if set to `true`, render all docstrings as collapsed by default.
```